docs/design/ftlib.md

```diff
@@ -19,32 +19,43 @@ Additionally, since most communication libraries require rank assigned to each w
 
 *Fig 2. The Role FTLib Played in Distributed Training*
 
-In the contemporary design, the FTLib is called explicitly to perform collective operation on all parameters of a model.
+In the contemporary design, the FTLib is called explicitly to perform collective operations or self-designed high-level calls.
```

Comment:
What do you mean by "self-designed high-level calls"?

Reply:
here I mean "wrapped high-level operations that executed when `FTLib.execute()` is called"